### DISCUSSION TITLE
What is difference between file formats?

### DISCUSSION THREAD

**Discussion by ghosharpita-ee** [deniscerri/ytdlnis#59] [Q&A] [answered]
![Screenshot_2023-03-18-23-48-05-32_2bde0e07fd61d625cf63c538621e2efc](https://user-images.githubusercontent.com/93784753/226126505-9fbcb536-8012-46fb-bd20-c67cb374b513.jpg)
As in the image, the file size for a video varies not only to the resolution also the file format. 

So my question is what will be the difference if i download same resolution but in different format?

**Comment by deniscerri** [comment, marked as answer]
MP4 / WEBM / MKV are just containers. To a normal person there isnt much of a difference.
H264 / VP9 / AV1 are encodings. The last two are newer formats and u will have better quality on them, although its not that big of a difference. But if you are downloading 1080p and up, choose them. 

I believe if you have an older / slower android VP9 might run slow on it but i haven't tested it.

**Reply by ghosharpita-ee** [reply]
Ohhh i get it. 
That's solves doubt
